waku/v2/utils/time.nim

```diff
@@ -0,0 +1,15 @@
+## Contains types and utilities for timestamps.
+
```

Comment:
I'd suggest defining a `Timestamp` type in this module (as an alias for int64), and replacing all the int64 types in the code with that. It makes it easier to keep track of all the timestamp values in the code and potentially makes future changes and refactoring easier.